baremetal-hello.s

```diff
@@ -29,6 +39,32 @@ _start:
         lx      t0, 0,(t0)
         bne     t0, s2, 1b
 
+        # Set the timer: read current time from mtime, add delta time to it,
+        # and write the result to mtimecmp:
+        li      t2, MTIME
+        ld      t2, 0(t2)               # read from mtime mmapped register
```

Comment:
Note, these 64-bit instructions break the RV32 build. I guess I'll factor it out into separate files which we can link into appropriate binaries.

Reply:
You can use .ifdef RISCV32 .else .endif

Btw, do you need to explicitly set MTIME address? I thought assembler should know mtime

Reply:
Good point, that's easy enough to do right away.